public/script.js

```diff
@@ -357,12 +357,20 @@ function getState(map, projection) {
       let totalDays = Math.abs(firstDate.diff(lastDate, 'days') + 1);
       let forwardDays = (totalDays / 4) * i;
       let scaleDate = moment(firstDate).add(forwardDays, 'd');
-      $(`#scale${i}`).html(scaleDate.format('YYYY'));
+      if (STATE.temporality == 'monthly') {
```

Comment:
Just to DRY things out a bit, I would be tempted to do something like this:

```
let date_format = 'YYYY-MM-DD'
if (STATE.temporality == 'monthly') {
  date_format = 'YYYY-MM'
}

$(`#scale${i}`).html(scaleDate.format(date_format));
```
or maybe even use a ternary operator:
```
$(`#scale${i}`).html(
  scaleDate.format(
    STATE.temporality == 'monthly'? 'YYYY-MM' : 'YYYY-MM-DD'
  )
);
```